pkgs/build-support/vm/default.nix

```
@@ -203,8 +205,8 @@ rec {
     if test -n "$origBuilder" -a ! -e /.debug; then
       exec < /dev/null
       ${coreutils}/bin/touch /.debug
-      $origBuilder $origArgs
-      echo $? > /tmp/xchg/in-vm-exit
+      $origBuilder ''${origArgs[*]}
```

I hope to preserve spaces in `args` when `__structuredAttrs = true`.

```suggestion
      declare -a argsArray=()
      concatTo argsArray origArgs
      $origBuilder "''${argsArray[@]}"
```

BTW, should we quote `$origBuilder`?